config/site/src/_includes/testimonials.html

```diff
@@ -7,27 +7,28 @@ <h2 class="text-3xl font-bold text-gray-900 dark:text-gray-100 mb-8">
     <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-8">
       <div class="bg-gray-100 dark:bg-gray-700 p-6 rounded-lg shadow">
         <p class="text-lg italic mb-4">
-          "ElasticGraph has revolutionized the way we manage and analyze our
-          data."
+          Just about everything has been thought of and you can go from prototype to fully
+          configured OpenSearch and GraphQL infra in a matter of days.
         </p>
         <div class="font-bold">
-          - John Doe, CTO at TechCorp
+          - Nicole W, Software Engineer at Block
         </div>
       </div>
       <div class="bg-gray-100 dark:bg-gray-700 p-6 rounded-lg shadow">
         <p class="text-lg italic mb-4">
-          "A must-have tool for any data-driven organization."
+          ElasticGraph enabled me to build a complex GraphQL API for our Payouts data in
+          days. It would have taken me months without it!
         </p>
         <div class="font-bold">
-          - Jane Smith, Data Scientist
+          - Max N, Software Engineer at Block
         </div>
       </div>
       <div class="bg-gray-100 dark:bg-gray-700 p-6 rounded-lg shadow">
         <p class="text-lg italic mb-4">
-          "ElasticGraph makes complex data operations simple and efficient."
+          Use ElasticGraph and love it? We'd love to hear from you! Send us a testimonial.
         </p>
         <div class="font-bold">
-          - Gandalf Skywalker, Software Engineer
```

Comment:
We should reach out to Gandalf Skywalker and see if he'll provide testimonial...